.eslintrc

```diff
@@ -14,6 +16,7 @@
     "import/no-named-as-default": 2,
     "comma-dangle": 0,  // not sure why airbnb turned this on. gross!
     "indent": [2, 2, {"SwitchCase": 1}],
+    "no-case-declarations": 0,
```

Comment:
actually I would not set this to 0 instead I would set `object-curly-spacing` to 0